server/db/pinSchema.js

```diff
@@ -21,26 +21,18 @@ const pinSchema = new mongoose.Schema(
   { strict: 'throw' }
 )
 
-const Pin = new mongoose.model('Pin', pinSchema)
+class PinClass {
+  static async findandRemoveOldPins() {
+    const ttl = process.env.TTL || 14
+    let cutoffDate = new Date()
```

Comment:
Just a small note - you can still use `const` here because using the `setDate` method is not reassigning the variable. `const` just means this variable can't be reassigned.